src/fury.es6

```diff
@@ -7,16 +8,17 @@ import legacyAPI from './legacy/blueprint';
 import legacyBlueprintParser from './legacy/blueprint-parser';
 import legacyMarkdownRenderer from './legacy/markdown';
 
-// Register Parse Result and API description primitives
-import './refract/parseResult';
+const minim = minimModule.namespace()
+  .use(minimParseResult);
```

Comment:
I agree. It was a bit easier with `require(...)` because you could do this:

```js
const minim = require('minim').namespace()
  .use(require('minim-parse-result'));
```

This was the best I came up with so far as the ES6 alternative. Again open to suggestions :grimacing: